uniconnect-epfl/uniconnect
# feat: add a button for a user to be able to register to an event

# What I did
- Added a component `BackArrow` and used it in screen to see details of annoucements and events.
- Added a button to participate to an event on `ViewEventDetails` screen.
- Slightly improved the styles for the view details screens.
- Adapted and added tests.

# How I did it
- Added a component `ArrowBack` in components and its corresponding test.
- Added the arrow back to `ViewEventDetails` and `ViewAnnouncementDetails` screens.
- Added a participate button on the `ViewEventDetails` screen.
- Slightly modified the styles in of `ViewEnventDetails` and also `ViewDetailsStyles`.

# How to verify it
Launch the app in Expo go and:
- Try to go on an event detail screen, check that the layout is good.
- Try the "Participate" button.
- Try the arrow back.
- Go on an announcement detail page, check that everything is good.
- You can also try the arrow back here.

# Demo video

https://github.com/uniconnect-epfl/uniconnect/assets/93329823/cbc69be6-4fef-4e69-9c7d-9e7c5e0b9727

# Pre-merge checklist
The changes I have introduced:
